Wikibase.php

```diff
@@ -160,6 +190,13 @@
 	$wgWBClientSettings['linkItemTags'] = [
 		'client-linkitem-change'
 	];
+	$wgWBClientSettings['sendEchoNotification'] = true;
+	$wgWBClientSettings['echoIcon'] = [
+		'url' => 'https://static.miraheze.org/commonswiki/a/a4/GDechoIcon.svg',
+	];
+	$wgWBClientSettings['allowLocalShortDesc'] = true;
+	$wgWBClientSettings['forceLocalShortDesc'] = true;
+	$wgWBClientSettings['showExternalRecentChanges'] = true;
```

Comment:
This is the default.

Reply:
Ah, no wonder I could see it from recent changes, but was still doubting.